File: Beecrowd/1016 Distância/Main.java
import java.io.IOException;
import java.util.Scanner;

public class Main {

    public static void main(String[] args) throws IOException {
        Scanner sc = new Scanner(System.in);

        final int X = 60, Y = 90;
        int distancia;

        distancia = sc.nextInt();

        distancia /= ((Y / 60.0) - (X / 60.0));

        System.out.println(distancia + " minutos");

        sc.close();
    }

}

File: Beecrowd/1016 Distância/main.c
#include <stdio.h>

int main() {

    int Distancia, X = 60, Y = 90;

    scanf("%d", &Distancia);

    Distancia = Distancia / ((Y / 60.0) - (X / 60.0));

    printf("%d minutos\n", Distancia);

    return 0;
}